docs/content/any/reference/polar/builtin_rule_types.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Built-in Rule Types
+any: true
+weight: 3
+description: |
+  Reference for built-in rule types.
+---
+
+# Built-in Rule Types
+
+Certain rules have special behavior in Oso. Built-in [rule types](reference/polar/polar-syntax#rule-types) define the expected arguments for special rules so that validation can be performed at policy load-time.
```

Comment:
Something weird is happening with these links when i'm running the docs locally. They go to the correct heading for a spit second then move to an incorrect heading and I'm not sure why.